docs/commands/eject.md

```diff
@@ -0,0 +1,92 @@
+# eject
+
+## Introduction
+
+The `eject` command allows you to customize components by creating a local copy of their source code in your Redocly project.
```

Comment:
Should we clarify that this is only for Revel, Reef and Realm?

Reply:
Yes. Added a sentence at end of intro with links. 

I'd like to standardize how we include this when posting in remote content sources.